src/FitterAlgoBase.cc

```diff
@@ -211,7 +211,8 @@ RooFitResult *FitterAlgoBase::doFit(RooAbsPdf &pdf, RooAbsData &data, const RooA
        std::cout << "Minimized in : " ; tw.Print();
     }
     nll0Value_ =  nll0;
-    nllValue_ =  nll->getVal() - nll0;
+    //nllValue_ =  nll->getVal() - nll0;
```

Comment:
We made this change because we needed in some cases to fix the likelihood manually. Is there harm in keeping this? Without this change, nll subtracts nll0 at initial values, which is not very useful IMO.